README.md

```diff
@@ -51,6 +51,18 @@ To disable a default schedule, just set `enabled` to `false`, e.g ``snapshot.hou
 
 The cleanup schedule is configured to run every hour by default. This will delete snapshots that are outside the keep-range of the schedules. This schedule can be changed via ``cleanup.cron`` setting
 
+### Time Zone
+
+All the schedules are using UTC timezone by default, however you can change it by adding 'timezone' property in the config file:
+
+    snapshot.snapshotname.timezone=GMT+02:00
```

Comment:
```suggestion
    snapshot.snapshotname.timezone=GMT+07:00
```

Reply:
FYI: It would be really nice with an example for a named location as well, like Europe/Oslo + a link to where all the available locations are listed.

Reply:
There is such example below, actually.

As for list, one, if not too lazy, can Google it easily:  https://en.wikipedia.org/wiki/List_of_tz_database_time_zones